src/frontend/package.json

```diff
@@ -27,41 +27,43 @@
         "@mantine/modals": "<7",
         "@mantine/notifications": "<7",
         "@naisutech/react-tree": "^3.1.0",
-        "@sentry/react": "^7.74.1",
-        "@tabler/icons-react": "^2.39.0",
-        "@tanstack/react-query": "^5.0.0",
-        "axios": "^1.6.0",
+        "@sentry/react": "^7.102.1",
+        "@tabler/icons-react": "^2.47.0",
+        "@tanstack/react-query": "^5.24.1",
+        "axios": "^1.6.7",
         "dayjs": "^1.11.10",
         "easymde": "^2.18.0",
         "embla-carousel-react": "^7.1.0",
         "html5-qrcode": "^2.3.8",
-        "mantine-datatable": "^6.0.5",
+        "mantine-datatable": "<7",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
-        "react-grid-layout": "^1.4.2",
-        "react-hook-form": "^7.48.2",
-        "react-router-dom": "^6.17.0",
-        "react-select": "^5.7.7",
+        "react-grid-layout": "^1.4.4",
+        "react-hook-form": "^7.50.1",
+        "react-is": "^18.2.0",
+        "react-router-dom": "^6.22.1",
+        "react-select": "^5.8.0",
         "react-simplemde-editor": "^5.2.0",
-        "styled-components": "^6.1.0",
-        "zustand": "^4.4.3"
+        "styled-components": "^5.3.6",
+        "zustand": "^4.5.1"
     },
     "devDependencies": {
-        "@babel/core": "^7.23.2",
-        "@babel/preset-react": "^7.22.15",
-        "@babel/preset-typescript": "^7.23.2",
-        "@lingui/cli": "^4.5.0",
-        "@lingui/macro": "^4.5.0",
-        "@playwright/test": "^1.39.0",
-        "@types/node": "^20.8.6",
-        "@types/react": "^18.2.28",
-        "@types/react-dom": "^18.2.13",
-        "@types/react-grid-layout": "^1.3.3",
+        "@babel/core": "^7.23.9",
+        "@babel/preset-react": "^7.23.3",
+        "@babel/preset-typescript": "^7.23.3",
+        "@lingui/cli": "^4.7.1",
+        "@lingui/macro": "^4.7.1",
+        "@playwright/test": "^1.41.2",
+        "@types/node": "^20.11.20",
+        "@types/react": "^18.2.59",
+        "@types/react-dom": "^18.2.19",
+        "@types/react-grid-layout": "^1.3.5",
         "@types/react-router-dom": "^5.3.3",
-        "@vitejs/plugin-react": "^4.1.0",
+        "@vitejs/plugin-react": "^4.2.1",
         "babel-plugin-macros": "^3.1.0",
-        "typescript": "^5.2.2",
-        "vite": "^4.5.2",
+        "typescript": "^5.3.3",
+        "vite": "^5.1.4",
         "vite-plugin-babel-macros": "^1.0.6"
-    }
+    },
+    "packageManager": "yarn@4.1.0"
```

Comment:
Yarn@v4 is very new to me, but I read a bit about it now. Seems like we were accidentally using yarn@v1 the whole time. And if we upgrade we need to also add how to enable core pack to the docs and test if the docker container build still works. And secondly as far as I read, core pack is currently experimental.

Reply:
I do not think I have enabled corepack (i just ran yarn set version) but validating it runs every without problems is certainly something that has to be done before merge.

Reply:
When I tried it out on macOS i got a message, that I first have to run `corepack enable`. I then removed the version constrain and the error disappeared.

Reply:
I have switched back to v1 now